bin/build

```diff
@@ -10,16 +10,24 @@ MIQ_ORG=${MIQ_ORG:-ManageIQ}
 SUI_ORG=${SUI_ORG:-ManageIQ}
 APPLIANCE_ORG=${APPLIANCE_ORG:-ManageIQ}
 
+REPO_PREFIX=${REPO_PREFIX:-manageiq}
+
+GIT_HOST=${GIT_HOST:-github.com}
```

Comment:
This should probably include the `https://` bit as well, in case people want to use something else like `ssh://`

Reply:
If we include the `https://` bit then building the url gets significantly more complicated. Maybe we could leave it like this for now and if we need to add support for a ssh key, we could refactor it then?